src/mesh/tdycoremeshutils.c

```diff
@@ -399,7 +399,7 @@ PetscInt TDyMeshGetNumberOfLocalFacess(TDy_mesh *mesh) {
 }
 
 /* -------------------------------------------------------------------------- */
-PetscInt TDyMeshGetNumberOfNonLocalFacess(TDy_mesh *mesh) {
+PetscInt TDyMeshGetNumberOfNonLocalFacess(TDyMesh *mesh) {
```

Comment:
Why are there two s's at the end of this function name? Just curious. Same goes for `TDyMeshGetNumberOfLocalFacess`